tests/gtests/CMakeLists.txt

```diff
@@ -18,15 +18,15 @@ add_subdirectory (gtest)
 
 set(APP_NAME "gtest")
 set(MAIN_SRC_GTEST main.cpp)
-include_directories(${CMAKE_SOURCE_DIR}/include)
-include_directories(${CMAKE_SOURCE_DIR}/tests/gtests
-                    ${CMAKE_SOURCE_DIR}/tests/gtests/gtest
+
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}
+                    ${CMAKE_CURRENT_SOURCE_DIR}/gtest
+                    ${CMAKE_CURRENT_SOURCE_DIR}/in
                     )
-include_directories(${CMAKE_SOURCE_DIR}/tests/gtests/in)
 # TODO: enable me!
 #file(GLOB API_TEST_CASES_SRC api_tests/*.cpp)
 #file(GLOB PRIM_TEST_CASES_SRC test_*.cpp)
-file(GLOB PRIM_TEST_CASES_SRC RELATIVE ${CMAKE_SOURCE_DIR}/tests/gtests
+file(GLOB PRIM_TEST_CASES_SRC RELATIVE ${CMAKE_CURRENT_SOURCE_DIR}/../gtests
```

Comment:
This change is incompatible with cmake 2.8.11.

For some reasons `PRIM_TEST_CASES_SRC = ../../test_sum.cpp;../../test_reorder.cpp;...` rather than `PRIM_TEST_CASES_SRC = test_sum.cpp;test_reorder.cpp;...`.

Is there any particular reason why you do this trick: ../gtests, even though you are already in this directory?